mail_parent_recipient/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Mail parent recipient",
+    "summary": "Sending the email in any case by the main partner",
```

Comment:
```suggestion
    "summary": ""Send email to parent partner if partner's email is empty"",
```
I'd still think using commercial partner instead of parent would make more sense.